cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -747,10 +747,11 @@ TEST_F(TestProjector, TestOffset) {
 
   // Create a row-batch with some sample data
   int num_records = 4;
-  auto array0 = MakeArrowArrayInt32({1, 2, 3, 4}, {true, true, true, true});
+  auto array0 = MakeArrowArrayInt32({1, 2, 3, 4, 5}, {true, true, true, true, true});
```

Comment:
can you please make some of these invalid ? that'll exercise the bitmap code too.